dist/api_data.py

```diff
@@ -566,6 +566,12 @@ def __ge__(self, other):
             operation. This allows memory sanitizers to detect inappropriate references to
             memory owned by cursors.''',
             type='boolean'),
+        Config('cursor_reposition', 'false', r'''
+            if true, for the operations with snapshot isolation the cursor temporarily releases the
+            page it would otherwise keep pinned, then repositions back to the page for further
```

Comment:
It only needs to release pages that require force eviction right?

Reply:
Updated in [a4aa307](https://github.com/wiredtiger/wiredtiger/pull/7977/commits/a4aa307e3659676a9ae22e4709e5bda62ff5927f). Thanks!